src/pages/guides/using/asset-events-providers.md

```diff
@@ -0,0 +1,73 @@
+# Asset Events Providers
+
+There are several different Asset Events providers (i.e., Adobe applications that generate events). Once you have created a project at https://developer.adobe.com/console and selected `Add Event`, you can select which events you want to receive from which providers. Below are all the event providers, along with their respective events.
+
+**Note**: Regardless of which provider is generating events, the structure and [properties](asset-events-properties.md) of the events remains consistent. 
+
+## Selecting an Event Provider
+
+![UI for selecting an event provider](../img/all_providers.png)
+
+The following applications are event providers:
+
+- Adobe Illustrator Cloud Docs
+- Creative Cloud Assets
+- Creative Cloud Libraries
+- Photoshop Cloud Docs
+- XD Cloud Docs
+
+Once you have selected a provider, you can subscribe to its respective events.
+
+## Adobe Illustrator Cloud Docs
```

Comment:
can we put this with the other cloud docs. in the order: 
Creative cloud assets
creative cloud libraries
and then AI cloud docs, PS and XD cloud docs.